mica/archive/aca_dark/dark_cal.py

```diff
@@ -9,12 +9,12 @@
 import pyyaks.context
 from Chandra.Time import DateTime
 
+from chandra_aca.dark_model import dark_temp_scale
```

Comment:
This should include `DARK_SCALE_4C`, the anneal pathfinder notebook imports from here.  Yes, useless and obscure, but there you go.